.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
     "files.associations": {
-        "ota_mqtt_private.h": "c"
+        "ota_mqtt_private.h": "c",
```

Comment:
is ota_mqtt_private.h required for this proof?

Reply:
Updated in 1b86b2f